cellmlmanip/model.py

```diff
@@ -555,21 +555,51 @@ def get_symbol_by_ontology_term(self, namespace_uri, local_name):
             raise ValueError('Multiple variables annotated with {%s}%s' %
                              (namespace_uri, local_name))
 
-    def _get_symbols_by_rdf(self, predicate, object_=None):
+    def get_rdf_annotation(self, predicate):
+        """Searches the RDF graph for rdf annotation with the given predicate
+        and (e.g. "pycml:named-attribute) and returns the associated annotation.
+
+        ``predicate`` must be a ``(namespace, local_name)`` tuple.
+        The method will return a dict following the same structure as the rdf annotation.
+        e.g. when looking for named-attribute (of which there could be multiple):
+        "{'model_name': {'named-attribute': [{'name': 'SuggestedForwardEulerTimestep', 'value': '0.002'}]}}"
+        Please note: for getting annotated model variables directly, use get_symbols_by_rdf
+        """
+        def _get_annotation(tuples):
```

Comment:
Call the argument `triples` for clarity.

Reply:
fair point